config/prod/veoibd-s3.yaml

```diff
@@ -1,7 +1,13 @@
 # Provision a general S3 bucket or a Synapse External Bucket
 # http://docs.synapse.org/articles/custom_storage_location.html
-template_path: "remote/s3-bucket.yaml"
+template:
+  type: "http"
+  url: "https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/aws-infra/v0.2.7/s3-bucket.yaml"
 stack_name: "veoibd-s3"
+stack_tags:
+  Department: "SysBio"
+  Project: "VEO-IBD"
+  OwnerEmail: "nicole.kauer@sagebase.org"
```

Comment:
No `CostCenter`?  Remember we have a "No Program" value for resources that are not directly funded.

Reply:
oops.  i'm assuming the OwnerEmail should change for this as well since nicole is no longer here.  Who would know the most appropriate person to own this now?  maybe they can also suggest a costcenter tag value.

Reply:
@zaro0508 put me first as maintainer then.  CostCenter can probably be determined by Larsson.

Reply:
changed OwnerEmail to @thomasyu888 and set CostCenter to "NO PROGRAM / 000000"